nautobot/docs/development/model-features.md

```diff
@@ -0,0 +1,40 @@
+# Registry Model Features
```

Comment:
I'm not opposed to splitting this out into a separate document, but note that there was some existing documentation of the `model_features` registry in `nautobot/docs/development/application-registry.md` that can now probably be removed and replaced with a link to this file?

Reply:
Sure it can be done